app/assets/src/styles/views/phylo_tree/phylo_tree_viz.scss

```diff
@@ -0,0 +1,105 @@
+.dropdown-menu {
+  background-color: #d3d3d3;
+  cursor: pointer;
+}
+
+.tree-selection-brush .extent {
+  fill-opacity: 0.05;
+  stroke: #fff;
+  shape-rendering: crispEdges;
+}
+
+.tree-scale-bar text {
+  font: sans-serif;
+}
+
+.tree-scale-bar line,
+.tree-scale-bar path {
+  fill: none;
+  stroke: #000;
+  shape-rendering: crispEdges;
+}
+
+.node circle,
```

Comment:
Can you add these bellow a tree's specific class, i.e. add a phylo-tree-view class to the div containing the tree? 
(add all the styling on this file as children of the class you use for the container div for you visualization).

For instace `.node circle` would be applied to all our d3 visualizations that have a node.

Reply:
I'll leave that for you in your UI work. Since I assume you'll replace the tree viz anyway